docs/general/attributes.md

```diff
@@ -400,4 +401,40 @@ about the span.
 | `code.column` | int | The column number in `code.filepath` best representing the operation. It SHOULD point within the code unit named in `code.function`. | `16` | Recommended |
 <!-- endsemconv -->
 
+## General geo attributes
+
+These attributes may be used to carry data about a specific location related to an event. 
+This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+
+<!-- semconv geo -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `geo.city_name` | string | City name. | `Montreal`; `Berlin` | Recommended |
+| `geo.continent_code` | string | Two-letter code representing continent’s name. | `AF` | Recommended |
+| `geo.continent_name` | string | Name of the continent. | `North America`; `Europe` | Recommended |
+| `geo.country_iso_code` | string | Two-letter ISO Country Code (ISO 3166-1 alpha2). | `CA` | Recommended |
+| `geo.country_name` | string | Country name. | `Canada` | Recommended |
+| `geo.location.lon` | double | Longitude of the geo location. | `-73.61483` | Recommended |
+| `geo.location.lat` | double | Latitude of the geo location. | `45.505918` | Recommended |
```

Comment:
Would recommend specifying WGS84 standard here to remove any ambiguity, e.g.
```suggestion
| `geo.location.lon` | double | Longitude of the geo location in WGS84. | `-73.61483` | Recommended |
| `geo.location.lat` | double | Latitude of the geo location in WGS84. | `45.505918` | Recommended |
```

Reply:
Done! Thanks!